gatsby/gatsby-ssr.tsx

```diff
@@ -21,6 +21,21 @@ export const onPreRenderHTML = ({
 		></script>
 	);
 
+	// Add Google Fonts
+	components.push(
```

Comment:
If we're changing our font loading mechanism from self-hosted with link preloads, then please could you remove the link headers from web.config as they're no longer needed?